tests/vm_test.go

```diff
@@ -693,6 +693,36 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				Should(Equal(k8sv1.ConditionFalse))
 		})
 
+		table.DescribeTable("[test_id:todo]should report an error status when VM scheduling error occurs", func(unschedulableFunc func(vmi *v1.VirtualMachineInstance)) {
```

Comment:
Don't forget `[test_id:todo]`

Reply:
I've requested a test_id to be allocated, waiting for it to arrive.
/hold

Reply:
@ILpinto can you assist in getting a test_id for this?

Reply:
I will be raising a PR with the test_id for this.

Reply:
@kbidarkar that would be great, thanks! :-)